replibyte/src/commands/dump.rs

```diff
@@ -145,7 +145,7 @@ where
                             password.as_str(),
                         );
 
-                        let task = FullDumpTask::new(postgres, datastore, options);
+                        let task = FullDumpTask::new(postgres, datastore, options, args.buffer_size);
```

Comment:
Hi @gugacavalieri , I would recommend putting `buffer_size` into the `options` struct instead of individually passing the parameter to the function. It's better to avoid touching the `FullDumpTask::new(..)` signature.